less/home-page.less

```diff
@@ -25,17 +25,32 @@
     display: flex;
     flex-wrap: wrap;
     width: 100%;
+    @media @phone {
+      justify-content: center;
+    }
```

Comment:
You could create one mixin called center having display flex and other centering properties of flex and use to.